pages/desktop/consumer_pages/account_settings.py

```diff
@@ -114,8 +117,15 @@ def edit_language(self, option_value):
         select = Select(element)
         select.select_by_value(option_value)
 
-    def toggle_recommendations(self):
-        self.selenium.find_element(*self._recommendations_checkbox_locator).click()
+    def disable_recommendations(self):
+        checkbox = self.selenium.find_element(*self._recommendations_checkbox_locator)
+        if not checkbox.is_selected():
+            raise AssertionError("Recommendations checkbox is expected to be checked, but it is unchecked")
```

Comment:
I would suggest calling `wait_for_recommended_tab_not_visible` here or move the contents into this method.